ansible/roles/test/templates/pfc_storm.j2

```diff
@@ -0,0 +1,5 @@
+enable
+configure terminal
+docker exec scapy "./pfcgen.py --p{{pfc_queue_index}} --q{{pfc_queue_index}}=65535 -i {{pfc_frames_number}} -d {{pfc_fanout_interface | replace("ernet 1/", "")}} -r {{ansible_eth0['ipv4']['address']}}"
```

Comment:
Is this the script we are going to use? I thought we will use pfctest.py. Where is the code to send syslog packets?

Reply:
what do you refer by code to send syslog packets? i am confused with the syslog and packets.

Reply:
We suppose to send syslog packets along with PFC frames to validate timer right? @marian-pritsak

Reply:
see https://github.com/marian-pritsak/pfctest/blob/master/pfctest.py#L302

Reply:
Thanks for the clarification. Since this template is fanout-hwsku dependent. Can we check the hwsku when using it?

Reply:
Done